sections/07_conclusions.md

```diff
@@ -52,32 +51,28 @@ Conversely, the most challenging tasks may be those in which predictions are
 used directly for downstream modeling or decision-making, especially in the
 clinic.  As an example, errors in sequence variant calling will be amplified if
 they are used directly for GWAS. In addition, the stochasticity and complexity
-of biological systems implies that for some problems, for instance, predicting
+of biological systems implies that for some problems, for instance predicting
 gene regulation in disease, perfect accuracy will be unattainable.
 
 We are witnessing deep learning models achieving human-level performance across
-a number of biomedical domains, and yet do not believe that biologists and
-clinicians will be displaced anytime soon. While deep learning methods will
-soon be (or already are) better than scientists at specific tasks, they may not
-fully grasp the bigger picture. Machine learning algorithms, including deep
+a number of biomedical domains. However, machine learning algorithms, including deep
 neural networks, are also prone to mistakes that humans are much less likely to
 make, such as misclassification of adversarial examples [@arxiv:1312.6199
 @arxiv:1412.6572], a reminder that these algorithms do not understand the
-semantics of the objects presented. Despite progress in addressing some of these
-limitations [@arxiv:1611.03814 @arxiv:1704.01155], we expect human experts will retain
-their critical roles in the laboratory and clinic
-for the foreseeable future. In some cases, cooperation
-between human experts and deep learning algorithms can achieve better
-performance than either individually [@arxiv:1606.05718]. Especially for sample
-and patient classification tasks, we expect deep learning methods to complement
-and assist biomedical researchers rather than compete with or even replace them.
+semantics of the objects presented. Work in this area is continuing
```

Comment:
I think the phrase "work in this area" should make it clear that the ongoing work is to guard against attacks and adversarial examples.  Maybe the second half of the sentence is clear enough though?